context-monitoring/resources/monitoring-config.xml

```diff
@@ -5,15 +5,21 @@
 
     <indexes>
```

Comment:
There seems to be an overlap between files here in `context-monitoring/resources` and `context-monitoring/src/test/resources`.

I assume only one location is actually needed?

Reply:
I am not sure why there are duplicated files in different locations. I have made changes in `context-monitoring/resources` and they seems to work. So may be we can delete the other one? @schlotze what do you say?

Reply:
`context-monitoring/resources` can be removed. AT least in the `dockerization` branch the configs for testing will be moved to `test/resources`

Reply:
Okay then I will suggest to keep both folders as it is now in this branch and then later remove one of them in `dockerization` branch.

Reply:
fine by me

Reply:
@PurvaPandyaVyas  ok. please add a TODO or comment to the respective files so that we don't forget.